app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -354,8 +354,9 @@ class MainPageActivity : AppCompatActivity(), OnRequestPermissionsResultCallback
         if(item.itemId==R.id.button_tutorial){
             viewPresentation(findViewById<View>(android.R.id.content).rootView)
         }
```

Comment:
Just for consistency I believe we use spaces between operators
```suggestion
        if(item.itemId == R.id.button_tutorial){
            viewPresentation(findViewById<View>(android.R.id.content).rootView)
        }
```